mlrun/datastore/base.py

```diff
@@ -369,7 +369,7 @@ def as_df(
         """return a dataframe object (generated from the dataitem).
 
         :param columns:   optional, list of columns to select
-        :param df_module: optional, dataframe class (e.g. pd, dd, cudf, ..)
+        :param df_module: optional, target dataframe class (e.g. pd, dd, cudf, ..)
```

Comment:
I think this message is a bit confusing. One might think it's a string.

Reply:
added `type`